inference-engine/tests/functional/inference_engine/lp_transformations/clamp_transformation.cpp

```diff
@@ -93,14 +93,14 @@ const std::vector<ClampTransformationTestValues> testValues = {
         LayerTransformation::createParamsU8I8(),
         // ActualValues
         {
-            ngraph::element::u8,
-            {{ngraph::element::f32}, {128.f}, {3.f}}
+            ngraph::element::Type(ngraph::element::u8),
+            {{ngraph::element::Type(ngraph::element::f32)}, {128.f}, {3.f}}
         },
         // ExpectedValues
         {
-            ngraph::element::u8,
+            ngraph::element::Type(ngraph::element::u8),
```

Comment:
one of the options to minimize a line is to create an inline function:
```c++
inline ngraph::element::Type ngraph::element::u8() {
    return ngraph::element::Type(ngraph::element::u8);
}
```
So, the only change in code would be adding `()` at the end. 

@tomdol @ilyachur how do you think? Just an idea..

Reply:
Sounds good to me too. It would require refactoring the code it a lot more places though, wouldn't it?

From my perspective it would make most sense to switch from `element::Type` to `element::Type_t` enum since most often we don't even need the information that the class provides. But this is an even bigger effort.

Reply:
If you want to reduce number of lines in this PR you may check if this works:
1) back to `enum class`
2) add `constexpr` to c-tor which take enum (and to enum cast operator)
3) add `constexpr` variables instead of `extern`

--
https://godbolt.org/z/nfGjh4